helix-term/src/ui/statusline.rs

```diff
@@ -142,6 +142,7 @@ where
         helix_view::editor::StatusLineElement::FileType => render_file_type,
         helix_view::editor::StatusLineElement::Diagnostics => render_diagnostics,
         helix_view::editor::StatusLineElement::Selections => render_selections,
+        helix_view::editor::StatusLineElement::TotalCharsSelected => render_total_chars_selected,
```

Comment:
The lingo is a little tricky for this feature: this statusline element will show the total number of characters selected _in the primary selection_. Could we call this `PrimarySelectionLength` or something along those lines so that we're specific about that?